ansible/group_vars/all.yml

```diff
@@ -89,6 +89,9 @@ origin_image_name: docker.io/openshift/origin
 origin_image_tag: v3.6.0-rc.0
 oc_host_config_dir: /var/lib/origin/openshift.local.config
 
+service_catalog_image_name: "origin-service-catalog"
+service_catalog_image_tag: "{{ origin_image_tag }}"
```

Comment:
They are used just for the syncing of the images.
The name of the image changes from:  origin-service-catalog to ose-service-catalog